futures-executor/src/spawn.rs

```diff
@@ -51,14 +51,58 @@ pub struct SpawnWithHandle<F>(Option<F>);
 /// onto the default executor. On completion, that future will yield a
 /// [`JoinHandle`](::JoinHandle) that can itself be used as a future
 /// representing the completion of the spawned task.
+///
+/// # Examples
+///
+/// ```
+/// # extern crate futures;
+/// # extern crate futures_executor;
+/// #
+/// use futures::prelude::*;
+/// use futures::future;
+/// use futures_executor::{block_on, spawn_with_handle};
```

Comment:
One question, should this be accessed via `futures::executor` instead of `futures_executor`? What is the policy on using the facade paths vs local paths in examples?

Reply:
I'd use the facade path to e consistent with the other examples.